packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -257,7 +275,7 @@ export const InnerProfileSettingsMenu = ({ className }: WithClassNameProps) => {
                   ...(isMobile && {
                     typography: {
                       type: TypographyType.Body,
-                      color: TypographyColor.Primary,
+                      color: TypographyColor.Secondary,
```

Comment:
Discussed with design team, this should be secondary. Reference: https://dailydotdev.slack.com/archives/C01473UNK9C/p1752654131811049